docs/src/code_base_developer_guide/troubleshooting.md

```diff
@@ -0,0 +1,22 @@
+# Troubleshooting code development
```

Comment:
This section lacks context.  Shouldn't `serialize` be in the heading?

Reply:
The first sentence doesn't make sense without context. I think this section should go back to where it was in the "add new struct" section.

Reply:
The idea is to make it a general section for TroubleShooting even if for now is only about serialization